apps/ledger-live-mobile/src/components/SelectDevice2/index.tsx

```diff
@@ -326,7 +329,7 @@ export default function SelectDevice({
           filterByDeviceModelId={filterByDeviceModelId}
         />
       ) : (
-        <Flex flex={1}>
+        <Flex flex={1} pb={insets.bottom + TAB_BAR_SAFE_HEIGHT}>
```

Comment:
Usually I would avoid using these safe area insets on components that can be integrated into screens, as the component doesn't have much info on the screen context, and maybe the parent screen is already adding this safe area itself, maybe it's already adding padding at the bottom, or maybe there is some content rendered below this component.
If you have checked all existing usages of the SelectDevice2 component and ensured it was ok then it's fine but otherwise please check, and as a general rule, setting "safe area" paddings like that should be the responsibility of the component that is rendered as a screen, so you ensure that only the "screen" component sets the safe area padding, and that its children don't also add a redundant padding.

Reply:
The way to go in my opinion would be to add a `paddingBottom` prop to the `SelectDevice2` component

Reply:
Added a `paddingBottom` props to the `SelectDevice2` that is effective conditionally, depending on `isPairingDevices` to avoid unnecessary padding during the error screen.